documentation/modules/deploying/con-securing-kafka-connect-api.adoc

```diff
@@ -0,0 +1,63 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deploy-kafka-connect-with-plugins.adoc
+
+[id='con-securing-kafka-connect-api-{context}']
+= Using the Kafka Connect API securely
```

Comment:
Technically, this is correct. But the reality is that we have a big gap around security here. So I wonder if we can have some more neutral title. Otherwise users will come and ask why does this say nothing about authentication, TLS etc.